Author: Daniel Baumann <daniel@debian.org>
Description: Ignoring stripped out documentation during build.

--- a/Makefile.am
+++ b/Makefile.am
@@ -43,7 +43,6 @@ if HAVE_NPAPI
 include $(srcdir)/plugin/Makefile.am.inc
 endif
 include $(srcdir)/test/Makefile.am.inc
-include $(srcdir)/doc/Makefile.am.inc
 
 EXTRA_DIST += zbar.ico zbar.nsi
 
--- a/Makefile.in
+++ b/Makefile.in
@@ -43,7 +43,7 @@ EXTRA_PROGRAMS = $(am__EXEEXT_1)
 DIST_COMMON = README $(am__configure_deps) $(am__dist_doc_DATA_DIST) \
 	$(am__zinclude_HEADERS_DIST) $(dist_man_MANS) \
 	$(include_HEADERS) $(srcdir)/Makefile.am $(srcdir)/Makefile.in \
-	$(srcdir)/doc/Makefile.am.inc $(srcdir)/gtk/Makefile.am.inc \
+	$(srcdir)/gtk/Makefile.am.inc \
 	$(srcdir)/include/Makefile.am.inc \
 	$(srcdir)/plugin/Makefile.am.inc \
 	$(srcdir)/pygtk/Makefile.am.inc \
@@ -53,7 +53,6 @@ DIST_COMMON = README $(am__configure_dep
 	$(srcdir)/zbar/Makefile.am.inc \
 	$(srcdir)/zbarcam/Makefile.am.inc \
 	$(srcdir)/zbarimg/Makefile.am.inc $(top_srcdir)/configure \
-	$(top_srcdir)/doc/doxygen.conf.in \
 	$(top_srcdir)/include/config.h.in COPYING ChangeLog INSTALL \
 	NEWS TODO config/compile config/config.guess \
 	config/config.rpath config/config.sub config/depcomp \
@@ -141,8 +140,6 @@ DIST_COMMON = README $(am__configure_dep
 @HAVE_MAGICK_TRUE@am__append_56 = test/dbg_scan
 @HAVE_GTK_TRUE@am__append_57 = test/test_gtk
 @HAVE_QT_TRUE@am__append_58 = test/test_qt
-@HAVE_MAGICK_TRUE@am__append_59 = doc/man/zbarimg.1
-@HAVE_VIDEO_TRUE@am__append_60 = doc/man/zbarcam.1
 @WIN32_TRUE@am__append_61 = README.windows
 subdir = .
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
@@ -157,7 +154,7 @@ am__CONFIG_DISTCLEAN_FILES = config.stat
  configure.lineno config.status.lineno
 mkinstalldirs = $(install_sh) -d
 CONFIG_HEADER = $(top_builddir)/include/config.h
-CONFIG_CLEAN_FILES = zbar.pc zbar-gtk.pc zbar-qt.pc doc/doxygen.conf
+CONFIG_CLEAN_FILES = zbar.pc zbar-gtk.pc zbar-qt.pc
 am__vpath_adj_setup = srcdirstrip=`echo "$(srcdir)" | sed 's|.|.|g'`;
 am__vpath_adj = case $$p in \
     $(srcdir)/*) f=`echo "$$p" | sed "s|^$$srcdirstrip/||"`;; \
@@ -678,7 +675,7 @@ BUILT_SOURCES = $(am__append_41) $(am__a
 # distribute all documentation related files to avoid end-user rebuilds
 EXTRA_DIST = $(am__append_33) $(am__append_39) $(am__append_43) \
 	$(am__append_48) test/test_pygtk.py test/test_perl.pl \
-	$(DOCSOURCES) $(man_stamp) doc/api/footer.html doc/style.xsl \
+	$(DOCSOURCES) $(man_stamp)  \
 	zbar.ico zbar.nsi examples/upcrpc.pl examples/upcrpc.py \
 	examples/paginate.pl examples/barcode.png \
 	examples/processor.pl examples/processor.py \
@@ -801,15 +798,14 @@ test_test_cpp_LDADD = zbar/libzbar.la $(
 @HAVE_QT_TRUE@test_test_qt_LDADD = $(QT_LIBS) qt/libzbarqt.la $(AM_LDADD)
 
 # documentation sources
-DOCSOURCES = doc/manual.xml doc/version.xml doc/reldate.xml \
-    doc/ref/zbarimg.xml doc/ref/zbarcam.xml doc/ref/commonoptions.xml
+DOCSOURCES = 
 
 
 # man page targets to distribute and install
 dist_man_MANS = $(am__append_59) $(am__append_60)
 
 # witness to man page build (many-to-many workaround)
-man_stamp = doc/man/man.stamp
+man_stamp = 
 # xmlto --searchpath broken again...
 doc_path = --searchpath $(abs_builddir)/doc -m \
 	$(abs_srcdir)/doc/style.xsl --skip-validation
@@ -823,7 +819,7 @@ all: $(BUILT_SOURCES)
 .SUFFIXES: .c .cpp .lo .o .obj
 am--refresh:
 	@:
-$(srcdir)/Makefile.in:  $(srcdir)/Makefile.am $(srcdir)/include/Makefile.am.inc $(srcdir)/zbar/Makefile.am.inc $(srcdir)/zbarimg/Makefile.am.inc $(srcdir)/zbarcam/Makefile.am.inc $(srcdir)/python/Makefile.am.inc $(srcdir)/gtk/Makefile.am.inc $(srcdir)/pygtk/Makefile.am.inc $(srcdir)/qt/Makefile.am.inc $(srcdir)/plugin/Makefile.am.inc $(srcdir)/test/Makefile.am.inc $(srcdir)/doc/Makefile.am.inc $(am__configure_deps)
+$(srcdir)/Makefile.in:  $(srcdir)/Makefile.am $(srcdir)/include/Makefile.am.inc $(srcdir)/zbar/Makefile.am.inc $(srcdir)/zbarimg/Makefile.am.inc $(srcdir)/zbarcam/Makefile.am.inc $(srcdir)/python/Makefile.am.inc $(srcdir)/gtk/Makefile.am.inc $(srcdir)/pygtk/Makefile.am.inc $(srcdir)/qt/Makefile.am.inc $(srcdir)/plugin/Makefile.am.inc $(srcdir)/test/Makefile.am.inc $(am__configure_deps)
 	@for dep in $?; do \
 	  case '$(am__configure_deps)' in \
 	    *$$dep*) \
@@ -877,8 +873,6 @@ zbar-gtk.pc: $(top_builddir)/config.stat
 	cd $(top_builddir) && $(SHELL) ./config.status $@
 zbar-qt.pc: $(top_builddir)/config.status $(srcdir)/zbar-qt.pc.in
 	cd $(top_builddir) && $(SHELL) ./config.status $@
-doc/doxygen.conf: $(top_builddir)/config.status $(top_srcdir)/doc/doxygen.conf.in
-	cd $(top_builddir) && $(SHELL) ./config.status $@
 install-libLTLIBRARIES: $(lib_LTLIBRARIES)
 	@$(NORMAL_INSTALL)
 	test -z "$(libdir)" || $(MKDIR_P) "$(DESTDIR)$(libdir)"
@@ -2615,9 +2609,8 @@ docs: $(dist_man_MANS) #dist_doc_DATA
 #doc/zbar-manual.pdf: $(DOCSOURCES)
 #	$(XMLTO) $(XMLTOFLAGS) -o doc pdf $<
 
-html-local: doc/html/index.html
+html-local: 
 doc/html/index.html: $(DOCSOURCES)
-	$(XMLTO) $(doc_path) $(XMLTOFLAGS) -o doc/html xhtml $<
 
 $(dist_man_MANS): $(man_stamp)
 	@if test ! -f $@ ; then \
@@ -2626,10 +2619,7 @@ $(dist_man_MANS): $(man_stamp)
 	fi
 
 $(man_stamp): $(DOCSOURCES)
-	@$(mkdir_p) doc/man 2>/dev/null
-	@rm -f $(man_stamp).tmp
 	@touch $(man_stamp).tmp
-	$(XMLTO) $(doc_path) $(XMLTOFLAGS) -o doc/man man $<
 	@mv $(man_stamp).tmp $(man_stamp)
 
 @WIN32_TRUE@%-rc.o: %.rc
--- a/configure
+++ b/configure
@@ -24059,7 +24059,7 @@ ac_config_commands="$ac_config_commands
 ac_config_commands="$ac_config_commands doc/reldate.xml"
 
 
-ac_config_files="$ac_config_files Makefile zbar.pc zbar-gtk.pc zbar-qt.pc doc/doxygen.conf"
+ac_config_files="$ac_config_files Makefile zbar.pc zbar-gtk.pc zbar-qt.pc"
 
 cat >confcache <<\_ACEOF
 # This file is a shell script that caches the results of configure
@@ -25290,7 +25290,6 @@ do
     "zbar.pc") CONFIG_FILES="$CONFIG_FILES zbar.pc" ;;
     "zbar-gtk.pc") CONFIG_FILES="$CONFIG_FILES zbar-gtk.pc" ;;
     "zbar-qt.pc") CONFIG_FILES="$CONFIG_FILES zbar-qt.pc" ;;
-    "doc/doxygen.conf") CONFIG_FILES="$CONFIG_FILES doc/doxygen.conf" ;;
 
   *) { { $as_echo "$as_me:$LINENO: error: invalid argument: $ac_config_target" >&5
 $as_echo "$as_me: error: invalid argument: $ac_config_target" >&2;}
--- a/configure.ac
+++ b/configure.ac
@@ -431,7 +431,6 @@ Makefile
 zbar.pc
 zbar-gtk.pc
 zbar-qt.pc
-doc/doxygen.conf
 ])
 AC_OUTPUT
 
